junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -826,6 +826,8 @@ private static void findNestedClasses(Class<?> clazz, Set<Class<?>> candidates)
 				.collect(toList());
 		// @formatter:on
 
+		localMethods.sort(ReflectionUtils::defaultMethodSorter);
+
```

Comment:
With respect to #13 I chose a more visible location. And that location offers the possibility to insert something like `Collections.shuffle(localMethods);` more easily.